### ISSUE TITLE
Click on title should close email

### ISSUE THREAD

**Issue by oscarryz** [boukestam/inbox-in-gmail#21]
Thanks for the chrome extension. 
Is it possible to add the feature where clicking on the email title closes the email?

**Comment by russelldc** [comment]
Interesting, I'll try working on this. In case you want a better experience in the meantime, I've turned on and customised the keyboard shortcuts in my Gmail settings, so I can hit the ESC key to return to the email list.

**Comment by joeytroy** [comment]
@russelldc I personally think keyboard shortcuts are faster than dealing with the mouse. Not sure the time to code this would be worth while.

**Comment by russelldc** [comment]
@g60madman Yeah, but I know a lot of people who would prefer to never move their hand from their mouse/touchpad. I don't completely understand it, but it's a significant amount of people.
